lib/WALASupport/SILWalaInstructionVisitor.cpp

```diff
@@ -721,88 +721,50 @@ jobject SILWalaInstructionVisitor::visitCondBranchInst(CondBranchInst *CBI) {
 }
 
 jobject SILWalaInstructionVisitor::visitSwitchValueInst(SwitchValueInst *SVI) {
-
   SILValue Cond = SVI->getOperand();
-  jobject CondNode = findAndRemoveCAstNode(Cond.getOpaqueValue()); 
+  jobject CondNode = findAndRemoveCAstNode(Cond.getOpaqueValue());
   
   if (Print) {
     llvm::outs() << "\t [COND]: " << Cond.getOpaqueValue() << "\n";
   }
 
-  SILBasicBlock *FalseBasicBlock = nullptr;
-  SILBasicBlock *TrueBasicBlock = nullptr;
-  
-  // Evaluate cases
+  // Make children
+  list<jobject> Children;
+
   for (unsigned Idx = 0, Num = SVI->getNumCases(); Idx < Num; ++Idx) {
     auto Case = SVI->getCase(Idx);
     auto *CaseVal = dyn_cast<IntegerLiteralInst>(Case.first);
```

Comment:
`CaseVal` should also be inserted to `Children`.

Reply:
Another thing I notice that the `Case.first` is not always an integer literal and we should not cast it here.